planner/core/cbo_test.go

```diff
@@ -246,14 +268,12 @@ func (s *testAnalyzeSuite) TestIndexRead(c *C) {
 	testKit.MustExec("create index e on t (e)")
 	testKit.MustExec("create index b_c on t (b,c)")
 	testKit.MustExec("create index ts on t (ts)")
-	for i := 0; i < 100; i++ {
-		testKit.MustExec(constructInsertSQL(i, 100))
-	}
 	testKit.MustExec("create table t1 (a int, b int, index idx(a), index idxx(b))")
+	err = s.loadTableStats("analyzesSuiteTestIndexReadT.json", dom)
```

Comment:
Where is the file?

Reply:
added

Reply:
Maybe we need to define this filename as a constant.
And add detailed comments for the statistics in it.

Reply:
Emmm why choose constant?

Reply:
I guess he means that we need to add some comments to illustrate how the statistics was generated.